### ISSUE TITLE
Autotype producing wrong characters on Windows with a custom Keyboard layout

### ISSUE THREAD

**Issue by leifb** [keeweb/keeweb#1795]
When I use autotype (either `ctrl+T` or `shift+alt+T`) into certain "targets", the produced characters do not match the actual information. I am on windows and am usually using the "United States-International" keyboard layout. When I switch to a different (German) keyboard layout, the autotype produces other characters, which are also wrong.

It used to work before I updated KeeWeb. It's been a while, but I believe I was on version `1.15.x` before, maybe `1.14.x`.

The behavior seems to be different depending on which application I type into: Autotyping into text editors and browsers works fine. I had the most problems with the Terminal of my Debian WSL. Powershell seems to leave out certain characters as well.

## To Reproduce

Create an entry and set `keystrokes` to only type `{USERNAME}`. Set the username to `type";?/\><äüö`. Autotype into any application.

Here is a side by side comparison of expected and actual outputs:

| Application | Debian WSL | Powershell
| --- | --- | --- |
| Expected | `type";?/\><äüö` | `type";?/\><äüö` 
| Actual (Int.) | `type@<{&\|\';` | `type";?/\><`
| Actual (Ger.) | `typeÄü'#^:;äüö` | `typeÄü'#^:;`

**Environment**
```
KeeWeb v1.17.6 (8efee7d, 2021-04-09)
Environment: electron v12.0.0
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.17.6 Chrome/89.0.4389.69 Electron/12.0.0 Safari/537.36
```

**Kdbx File**
[Kdbx File](https://github.com/leifb/keeweb-demo-file/raw/main/Autotype_layout.kdbx)

**Comment by antelle** [comment]
Hi! Does it work in other apps with the same layout, for example, in browser?

**Comment by leifb** [comment]
I you want I can test a few other applications, but these are the only ones I use.

**Comment by antelle** [comment]
Yeah I know why this happens, it's because some applications don't accept unicode input, I'll see what ca be done there.
